### ISSUE TITLE
How do I train a custom dataset

### ISSUE THREAD

**Issue by joryoker** [MediaBrain-SJTU/RegAD#6]
The pre-train models ave_checkpoints and support_set are based on MVTec dataset, How do I train a custom dataset with pre-train models?

**Comment by chaoqinhuang** [comment]
Please refer to [#1 ](https://github.com/MediaBrain-SJTU/RegAD/issues/1) for the guidance to make the support set for your custom data, and then in the few-shot setting, you can evaluate using your test set with the checkpoints. 
If you want to re-train the model with your additional training data, please re-organize your data as the same structure as MVTec and add them as another category, jointly training with MVTec.
